Libraries/LibWebView/Settings.cpp

```diff
@@ -336,14 +337,17 @@ Optional<SearchEngine> Settings::parse_custom_search_engine(JsonValue const& sea
 
     auto name = search_engine.as_object().get_string(search_engine_name_key);
     auto url = search_engine.as_object().get_string(search_engine_url_key);
+    auto bang = search_engine.as_object().get_string(search_engine_bang_key);
     if (!name.has_value() || !url.has_value())
         return {};
 
     auto parsed_url = URL::Parser::basic_parse(*url);
     if (!parsed_url.has_value())
         return {};
 
-    return SearchEngine { .name = name.release_value(), .query_url = url.release_value() };
+    auto bangVal = bang.has_value() ? bang.release_value() : Optional<String> {}; // this doesnt feel like the best way
+
+    return SearchEngine { .name = name.release_value(), .query_url = url.release_value(), .bang = bangVal };
```

Comment:
I think you can just do `SearchEngine { /* ... */ .bang = move(bang) }` and don't need `bangVal` at all, unless I'm missing something.

Reply:
OK. Then please just rename `bangVal` to match our conventions.

You can probably use `bang.map()` to convert from `String const&` to `String`, that might be cleaner. But what you have is fine too. No need for the comment.